### PR TITLE
Private/gokay/copy yy

### PR DESCRIPTION
**Pull request by gokaysatir**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by vmiklos — APPROVED** [review]
This reads nicely to me, I like how the new code can't forget to escape HTML. Thanks!

File: browser/src/map/Clipboard.js [hunk 1]
```diff
@@ -1268,7 +1306,31 @@ L.Clipboard = L.Class.extend({
 		var box = document.getElementById(id + '-box');
 		var innerDiv = L.DomUtil.create('div', '', null);
 		box.insertBefore(innerDiv, box.firstChild);
-		innerDiv.innerHTML = msg;
+
```
Comment (timar) [inline browser/src/map/Clipboard.js:1309]: I'm sorry that I joined late to the party. This chunk of the code is sub-optimal from the localization perspective. It is not possible to localize sentences broken in two pieces. 

Part 1: Please press
Part 2: to see more options

E.g. how to translate "Please press", if in some language the sentence translates to "To see more options please press <something>". Impossible.

Comment (vmiklos) [reply browser/src/map/Clipboard.js:1309]: @gokaysatir would it be reasonable to revert just this part *and* add a comment that content is built this way for l10n reasons, so the next person won't do the same mistake? Thanks.

Comment (gokaysatir) [reply browser/src/map/Clipboard.js:1309]: Well, this makes the issue complicated :) I will find another solution that satisfies both l10 and CSP sides.

**Pull request merged**